src/crudeditor-lib/views/search/container.js

```diff
@@ -26,60 +41,83 @@ import {
 } from './actions';
 
 const mergeProps = (
-  { defaultNewInstance, viewModelData, viewState, operations, externalOperations, uiConfig },
-  { softRedirectView, ...dispatchProps },
+  {
+    defaultNewInstance,
+    viewModelData,
+    viewState,
+    externalOperations,
+    customOpsConfig,
+    standardOpsConfig,
+    uiConfig
+  },
+  {
+    softRedirectView,
+    deleteInstances,
+    resetFormFilter,
+    searchInstances,
+    ...dispatchProps
+  },
   ownProps
 ) => ({
   ...ownProps,
   viewModel: {
     data: viewModelData,
     actions: {
-      ...dispatchProps,
-      createInstance: _ => softRedirectView({
-        name: VIEW_CREATE,
-        state: {
-          predefinedFields: defaultNewInstance
-        }
-      }),
-      editInstance: ({
-        instance,
-        tab,
-        index // an index of instance in the array of search results => add navigation to Edit View.
-      }) => softRedirectView({
-        name: VIEW_EDIT,
-        state: {
-          instance,
-          tab,
-          navigation: {
-            offset: viewModelData.pageParams.offset + index,
-            totalCount: viewModelData.totalCount
-          }
-        }
-      }),
-      showInstance: ({
-        instance,
-        tab,
-        index // an index of instance in the array of search results => add navigation to Show View.
-      }) => softRedirectView({
-        name: VIEW_SHOW,
-        state: {
-          instance,
-          tab,
-          navigation: {
-            offset: viewModelData.pageParams.offset + index,
-            totalCount: viewModelData.totalCount
-          }
-        }
-      })
+      ...dispatchProps
     },
     operations: {
-      internal: viewOperations({
+      custom: customOperations({
         viewName: VIEW_NAME,
         viewState,
-        operations,
+        operations: customOpsConfig,
         softRedirectView
       }),
-      external: externalOperations
+      external: externalOperations,
+      standard: standardOperations({
```

Comment:
I do not see a reason for additional layer of **operations.standardOperations**.